sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -691,7 +691,9 @@ case class Pivot(
     pivotColumn: Expression,
     pivotValues: Seq[Literal],
     aggregates: Seq[Expression],
-    child: LogicalPlan) extends UnaryNode {
+    child: LogicalPlan,
+    groupByExprsImplicit: Boolean = false) extends UnaryNode {
```

Comment:
Could you add add the parameter descriptions like what we did in `GroupingSets ` (line 663-675)?

Reply:
Could we just change `groupByExprs: Seq[NamedExpression]` -> `groupByExprs: Option[Seq[NamedExpression]]`? In the pattern matching, we just need to do something like Some, or None.